WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -2646,7 +2597,10 @@ title_royal_builder = {
 	}
 
 	allowed_to_grant = {
-		is_feudal = yes
+		OR = {
+			is_feudal = yes
+			is_merchant_republic = yes
+		}
```

Comment:
@loup99 

I disagree. I interpreted it as a royal architect of sorts: Merchant Republics build Trade Posts and Mansions, as well as upgrades to cities. The immense wealth a Republic gets should be administered by a minor title for flavor's sake. That said, it may have made more sense to use the Royal Architect title for this. If you'd like, I'll change it to that.